cmd/vclusterctl/cmd/import.go

```diff
@@ -25,15 +25,16 @@ func NewImportCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
 		Log:         log.GetInstance(),
 	}
 
-	description := `########################################################
-############### vcluster platform import ###############
-########################################################
+	description := `
+###############################################
```

Comment:
This will probably have to be one line iirc

```suggestion
	description := `###############################################
```